docs/customization/controller.rst

```diff
@@ -109,57 +110,67 @@ getting a list of recommended products from your external api.
 How to customize a Standard Controller:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Let's assume that you would like to send some kind of emails (which are not resources) after something has been purchased in your shop - to do this you should modify an ``afterPurchaseAction`` on the ``OrderController``.
+Let's assume that you would like to add some logic to the Homepage.
 
-1. Create a new Controller class under the ``AppBundle/Controller/Frontend`` namespace.
+1. Create a new Controller class under the ``AppBundle/Controller/Shop`` namespace.
 
-Run ``$ php bin/console debug:container sylius.controller.frontend.order``.
+Run ``$ php bin/console debug:container sylius.controller.shop.homepage``.
 
-Your class needs to extend this base class.
+Your class needs to extend the Symfony's Controller, and if you still need the methods of the original HompageController,
```

Comment:
Can we write something like: `We encourage you to define the controllers as a service`. And also there is no need of extending Symfony's Controller